resources/user-agents/browsers/safari/safari-10-0-on.json

```diff
@@ -30,15 +30,15 @@
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_12", "OSX_10_11", "OSX_10_10",
+            "OSX_10_12", "OSX_10_11", "OSX_10_10", "OSX_10_9",
```

Comment:
So, for the Safari on OSX platform versions I used this wikipedia article: https://en.wikipedia.org/wiki/Safari_version_history#Mac to try to limit the OSX versions to just what's in that table, but to be a bit conservative, I went +1 and -1 of the versions in that table to account for pre-release versions and perhaps cases where a new version of OSX would be using the previous version of Safari (this does seem like it would be rare though).

Open to discussion about it. Like the iOS version stuff I mentioned earlier, this won't cause any "Default Browsers" situations, as the generic platforms will still catch the agent, it just won't have a platform version.